src/loader/bgw_launcher.c

```diff
@@ -321,13 +321,25 @@ start_db_schedulers(HTAB *db_htab)
 
 		worker_registered = register_entrypoint_for_db(current_entry->db_oid, vxid, &current_entry->db_scheduler_handle);
 		if (!worker_registered)
-			report_error_on_worker_register_failure();
-
+		{
+			hash_seq_term(&hash_seq);
+			ereport(LOG, (errcode(ERRCODE_INSUFFICIENT_RESOURCES),
+						  errmsg("TimescaleDB background worker scheduler for at least one database unable to start."),
```

Comment:
`errmsg` should not end with period. See here: https://www.postgresql.org/docs/current/static/error-style-guide.html